docs/en/relying-party-solution.rst

```diff
@@ -21,13 +18,19 @@ In this section the following flows are described:
 
 In the **Same Device** and **Cross Device** Flows described in this chapter, the User interacts with a remote Relying Party.
 
+.. note::
+    The provisioning of the Wallet Instance Attestation from the Wallet Instace to the Relying Party is 
+    under discussion within the international standardization working groups. At the current stage of the draft of this implementation profile,
+    a mechanism based on `OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer (DPoP) <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop>`_
```

Comment:
Update it to RFC9449

Reply:
it redirects automatically to the last version (RFC)